### ISSUE TITLE
Origin animation distance

### ISSUE THREAD

**Issue by amingarro** [morellodev/react-awesome-reveal#86]
Please, could you add this feature?

Example: **distance="50px"**
`<Fade direction="right" distance="50px" cascade="true" damping="0.3">`

Thank you very much for your work, greetings.

**Comment by morellodev** [comment]
Hi, you can only play with the `fraction` prop listed [here](https://github.com/morellodev/react-awesome-reveal#props), because this is the way the Intersection Observer API works.

Anyway, be sure to pass the props using the right types:

```jsx
<Fade direction="right" cascade={true} damping={0.3}>
```

**Comment by nagman** [comment]
I don't think it's what @amingarro meant. `distance` here is relative to how much the element should translate during its animation.
The default is a translate of `100%`, but that's way too much in most cases.

**Comment by 000Erick** [comment]
I have the same problem I need the property "distance", is there any temporary solution ?

**Comment by yang** [comment]
You'll need to use custom keyframes, code snippet here: https://github.com/morellodev/react-awesome-reveal/issues/43

**Comment by josuearrieta** [comment]
Thoughts on adding this natively in the main component? `Fade` and `Slide` ?

**Comment by SteveAtKlover** [comment]
yes, this is needed. please add asap. should be simple to implement.
